docs/tutorials/jokes.md

```diff
@@ -5553,37 +5553,27 @@ import type { LoaderFunction } from "@remix-run/node";
 
 import { db } from "~/utils/db.server";
 
-function escapeCdata(s: string) {
```

Comment:
Why did those got removed? We definitely want to sanitize our RSS output.

Reply:
the file `jokes[.]rss.tsx` didn't have that.
But I'll add it in the example and back in this file.